.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "singleQuote": true,
+  "semi": false,
```

Comment:
I think this semi colon attribute should be set to true. It looks like it's causing edits in every file due to it removing semi colons